### ISSUE TITLE
Support `public.skipExportGlyphs` in `font.info`

### ISSUE THREAD

**Issue by benkiel** [adobe-type-tools/afdko#1082]
Could `makeotf` support the `public.skipExportGlyphs` in `font.info`? See discussion of it here: https://github.com/unified-font-object/ufo-spec/pull/84 (and, yes, we need to add this to `defcon` still...)

**Comment by josh-hadley** [comment]
Could you elaborate a bit on how you'd like to see this supported in `makeotf` (or elsewhere in AFDKO)?

**Comment by benkiel** [comment]
@josh-hadley Sorry for the slow reply. In brief, it would be great if `makeotf` would look for `public.skipExportGLyphs` if it ingests a UFO and if present it removes those glyphs from the generated OTF after decomposing the UFO.

**Comment by frankrolf** [comment]
I am curious how this would correspond with the GlyphOrderAndAliasDB’s `-gs` option (which excludes glyphs not mentioned within).
Which of the two methods would take precedence?

**Comment by benkiel** [comment]
Hmmmmm, that's a fair point. Use case for this in makeOTF is likely low, so feel free to close.

**Comment by miguelsousa** [comment]
> Which of the two methods would take precedence?

The option would take precedence over the key. I think it's the approach we've taken in other comparable cases. See https://github.com/adobe-type-tools/afdko/issues/765#issuecomment-471772510

**Comment by josh-hadley** [comment]
If/when we implement this, we should be sure to document which takes precedence and maybe issue a warning at runtime when both the `public.skipExportGlyphs` key and GOADB with `-gs` option are specified to avoid users making the wrong assumption.

**Comment by twardoch** [comment]
My view is that since skipExportGlyphs travels with the font source, it really is informative about what constitutes the "full glyph repertoire" of the final font. The glyphs in skipExportGlyphs are simply "not intended" to be included in the binary font at all. 

The -gs option is a positive filter for creating subsets, kind of equivalent to pyftsubset. 

But I agree that the -gs option should take precedence and "turn off" skipExportGlyphs because there is a valid use case that you may want to include the otherwise skipped glyphs in GOADB. 

Basically, in a UFO worfklow, GOADB is a mechanism for overriding specificatiins found in the UFO — glyph set, glyph names Unicode codepoint assignments. Right?
